packages/vendure-plugin-shipmate/test/shipmate.spec.ts

```diff
@@ -104,7 +109,33 @@ describe('Shipmate plugin', async () => {
 
   let order: Order | undefined;
 
+  it('Should not create a Shipment when an Order.totalQuantity is >= 5', async () => {
+    nock(nockBaseUrl)
+      .post('/tokens', (reqBody) => {
+        return true;
+      })
+      .reply(200, {
+        message: 'Login Successful',
+        data: {
+          token: '749a75e3c1048965c498017efae8051f',
+        },
+      })
+      .persist(true);
+    let shipmentRequest: any;
```

Comment:
This shipmentRequest is never assigned in the Nock request, so this test will always pass. The test below is doing it correctly: `shipmentRequest = reqBody;`